app/Image.php

```diff
@@ -34,4 +34,21 @@ public function getImageUrlAttribute()
     {
         return asset(config('define.images_path_products') . $this->image);
     }
+
+    /**
+    * Get the path product's image.
+    *
+    * @return string
+    */
+    public function toArray()
```

Comment:
dùng append `https://laravel.com/docs/5.6/eloquent-serialization#appending-values-to-json` để khỏi viết lại chổ này